templates/demo-store/app/routes/($locale).products.$productHandle.tsx

```diff
@@ -59,7 +59,7 @@ export async function loader({params, request, context}: LoaderArgs) {
   }
 
   if (!product.selectedVariant) {
-    return redirectToFirstVariant({product, request});
+    throw redirectToFirstVariant({product, request});
```

Comment:
This makes returned types work better, and I think it doesn't change the behavior, right?